CMakeLists.txt

```diff
@@ -268,6 +268,18 @@ if(UNIX AND NOT APPLE)
   endif()
 endif()
 
+# Check for SELinux library
+if(UNIX AND NOT APPLE)
+  check_include_files(selinux/selinux.h HAVE_SELINUX_H)
+  set(CMAKE_REQUIRED_LIBRARIES -lselinux)
+  set(CMAKE_REQUIRED_LIBRARIES)
+  if(HAVE_SELINUX_H)
+    set(SELINUX_LIBS selinux)
+  else()
+    message(FATAL_ERROR "Could not find SELinux development files")
+  endif()
```

Comment:
Is this a reasonable hard requirement? Most distributions are not using SELinux.